.travis.yml

```diff
@@ -26,3 +26,5 @@ after_failure:
     - ${PULUMI_SCRIPTS}/ci/upload-failed-tests
 notifications:
     webhooks: https://ufci1w66n3.execute-api.us-west-2.amazonaws.com/stage/travis
+env:
+    - TF2PULUMI_VERSION=0.5.0
```

Comment:
This ensures we use the older tf2pulumi which did not treat data sources as synchronous by default.  We need to update our dependency on pulumi-terraform to remove this line, but it's not clear how to do that anymore, since this repository does not use go modules /cc @jen20 @stack72